.jshintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "quotmark": "single",
+  "node": true,
+  "globals": {
+    "angular": false,
+    "document": false,
+    "jasmine": false,
+    "module": false,
+    "describe": false,
+    "beforeEach": false,
+    "afterEach": false,
+    "it": false,
+    "inject": false,
+    "expect": false,
+    "spyOn": false
+  }
+}
```

Comment:
But if you really wanted to keep it in the same file, don't hesitate to
tell me. I could change it :)

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/huei90/angular-validation/pull/147/files#r42940384>.
>